docs/dg/dev/third-party-module-development/ensure-quality.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Ensure Quality
+description: Ensure Quality
+last_updated: Jun 7, 2024
+template: howto-guide-template
+related:
+  - title: Go to the Next Step.
+    link: docs/dg/dev/third-party-module-development/publish-code.html
+---
+
+<style>
+  table th, table td {
+    white-space: nowrap;
+  }
+</style>
+
+**TBD: Commands must be updated after https://spryker.atlassian.net/browse/FRW-8436 is done (Check if we need it at all)  **
+
+## Tools
+
+To make sure that code quality matched the Spryker standards it is important to execute code quality tools, here is the list:
+
+| Command                                                                                 | Description                                                             |
+|:----------------------------------------------------------------------------------------|:------------------------------------------------------------------------| 
+| console code:propel:validate-abstract -m YourCompanyName.YourModuleName                 | Validates that propel entity/query classes are implemented properly.    |
```

Comment:
Maybe we could also `composer validate` if this is not already documented somewhere? One of the most important features of this use case would be to validate that all recommended/required are set in the `composer.json`.

Reply:
Updated list of commands to have only those that work in standalone mode, we'll extend this list in the future when finish the development of related functionality.